tests/config/bug_633.cfg

```diff
@@ -1 +1,8 @@
+indent_columns=4
+indent_with_tabs=0
 sp_arith=add
+indent_class=true
+#
+# bug 633
+#
+include "MS-calling_conventions.cfg"

```

Comment:
i would call this msvc.cfg instead. config file would be better to apply to the entire compiler, rather than a subset of its keywords.